cloudmanager/projects.go

```diff
@@ -28,7 +28,7 @@ const (
 
 // ProjectsService is an interface for interfacing with the Projects
 // endpoints of the MongoDB Atlas API.
-// See more: https://docs.atlas.mongodb.com/reference/api/projects/
+// See more: https://docs.cloudmanager.mongodb.com/reference/api/groups/
```

Comment:
This interface appears to be using the atlas api?

Reply:
what do you mean by this? the link was a copy/paste from the atlas client that I'm correcting now

Reply:
Talked offline

Reply:
discussed offline, comment above needs to be fixed as well

Reply:
Fixed